### PR TITLE
Run notebook on Colab

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by jason-dai** [comment]
@dding3 please a look

File: docs/docs/ProgrammingGuide/run-notebook-colab.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Summary
+
+With colaboratory, we can easily setup and run code in the cloud. How can we run Analytics Zoo [tutorials](https://github.com/intel-analytics/zoo-tutorials) or examples on colaboratory?
```
Comment (dding3) [inline docs/docs/ProgrammingGuide/run-notebook-colab.md]: Maybe we can put a colaboratory link here in case user want more details about it.

Comment (Le-Zheng) [reply docs/docs/ProgrammingGuide/run-notebook-colab.md]: Colaboratory official link is added.

Comment (Le-Zheng) [reply docs/docs/ProgrammingGuide/run-notebook-colab.md]: Is there anything else that needs to be improved?

Comment (dding3) [reply docs/docs/ProgrammingGuide/run-notebook-colab.md]: LGTM

**Pull request merged**